### DISCUSSION TITLE
nhits example with futr_exog_list, hist_exog_list, stat_exog_list

### DISCUSSION THREAD

**Discussion by FarhangAmaji** [Nixtla/neuralforecast#681] [Q&A]
I want to know how these variables in nhits model futr_exog_list, hist_exog_list, stat_exog_list behave? I dont know which dataset, which is also multivariate, has these variables. the example in `LongHorizon_with_NHITS.ipynb` seems to be multivariate but not with mentioned variables. the other thing is currently I have problem with `ray` and it does not get initiated on my computer, so I cannot see where it splits the ettm2 cols to different cols. therefore I can't debug line by line through the code to see which shapes do matrices have in forward phase.
so if there is any example for multivariate nhits with a dataset which has those variables and its not using `ray` is appreciated.

**Comment by kdgutier** [comment]
Hey @FarhangAmaji,

Thanks for using NeuralForecast.
1. Here is an example of exogenous variables usage: https://nixtla.github.io/neuralforecast/examples/exogenous_variables.html
2. As we explain in the discussion of Section 5 in the [NHITS paper](https://arxiv.org/pdf/2201.12886.pdf), we don't treat the long-horizon datasets as multivariate. NHITS only takes the series' own autoregressive features for predictions.
3. It is advisable to use GPUs to run the experiments. I recommend using Google Colab; see the tutorial links.
4. Here is an example of NHITS not using ray: https://nixtla.github.io/neuralforecast/examples/robust_regression.html
